mcgill-augmented-reality-backend/src/main/java/mcgillar/backend/controllers/image/ImageDataController.java

```diff
@@ -19,28 +19,56 @@ public class ImageDataController {
 
     private ImageDataService imageDataService;
 
-    @PostMapping
-    public ResponseEntity<?> uploadImage(@RequestParam("image") MultipartFile file) throws IOException {
+    @PostMapping("schedule")
+    public ResponseEntity<?> uploadScheduleImage(@RequestParam("image") MultipartFile file) throws IOException {
         String name = SecurityContextHolder.getContext().getAuthentication().getName();
 
-        imageDataService.uploadImage(file, name);
+        imageDataService.uploadImage(file, "schedule_" + name);
 
         return new ResponseEntity(HttpStatus.OK);
     }
 
-    @GetMapping("info/")
-    public ResponseEntity<?>  getImageInfoByName(){
+    @PostMapping("profile")
+    public ResponseEntity<?> uploadProfileImage(@RequestParam("image") MultipartFile file) throws IOException {
         String name = SecurityContextHolder.getContext().getAuthentication().getName();
-        ImageData image = imageDataService.getInfoByImageByName(name);
+
+        imageDataService.uploadImage(file, "profile_" + name);
+
+        return new ResponseEntity(HttpStatus.OK);
+    }
+
```

Comment:
Can we add some put methods for the 2 methods above so we can update the image in the db if it already exists? Currently, we just save it to the db along with the others so when we get them by the username it returns more than one which i believe is causing an error on the backend.